src/foam/u2/view/UnstyledTableView.js

```diff
@@ -418,16 +418,25 @@ foam.CLASS({
                   return;
 
                 var tableWidth = view.columnHandler.returnPropertyForColumn(view.props, view.of, [ col, overrides], 'tableWidth');
+                var colTitle = view.columnConfigToPropertyConverter.returnColumnHeader(view.of, col);
+                var colHeader = (colTitle.length > 1 ? '../'  : '') + colTitle.slice(-1)[0];
 
                 this.start().
                   addClass(view.myClass('th')).
                   addClass(view.myClass('th-' + prop.name))
-                  .style({ flex: tableWidth ? `0 0 ${tableWidth}px` : '1 0 0', 'word-wrap' : 'break-word', 'white-space' : 'normal'})
-                  .start()
+                  .style({
+                    'align-items': 'center',
+                    'display': 'flex',
```

Comment:
No need to quote single work keys like display, flex, and overflow.

Reply:
Just makes it more consistent with surrounding code. But I can change that